mesheryctl/internal/cli/root/environments/create.go

```diff
@@ -84,7 +84,10 @@ https://docs.layer5.io/cloud/spaces/environments/
 			return err
 		}
 
-		resp, _ := utils.MakeRequest(req)
+		resp, err := utils.MakeRequest(req)
+		if err != nil {
+			return err
```

Comment:
Can you use a meshkit errors instead ref: https://docs.meshery.io/project/contributing/contributing-error